.github/workflows/gcp-deploy.yml

```diff
@@ -36,4 +37,4 @@ jobs:
 
     - name: Deploy Docker utmap-server
       run: |
-        gcloud run deploy utmap-server --image $IMAGE_NAME --region us-central1 --platform managed --allow-unauthenticated --set-env-vars="SERVER_HOST=0.0.0.0" --set-env-vars="SERVER_PORT=8080"
+        gcloud run deploy utmap-server --image $IMAGE_NAME --region us-central1 --platform managed --allow-unauthenticated --set-env-vars="SERVER_HOST=0.0.0.0" --set-env-vars="SERVER_PORT=8080" --set-env-vars="SKIP_DB_INIT=${{SKIP_DB_INIT}}" --set-env-vars="MONGODB_HOST=${{secrets.MONGODB_HOST}} --set-env-vars="MONGODB_USERNAME=${{secrets.MONGODB_USERNAME}} --set-env-vars="MONGODB_PASSWORD=${{secrets.MONGODB_PASSWORD}}"
```

Comment:
> --set-env-vars="SKIP_DB_INIT=${{SKIP_DB_INIT}}"

`${{}}` is used to access secrets, not environment variables, which is what `SKIP_DB_INIT` is. Have a look at how `IMAGE_NAME` is referenced, since its also an environment variable.